docker/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
```

Comment:
ATLAS requires the WebAPI url to end with a '/', so we could add a small check here:

```suggestion
# make sure the WebAPI URL ends with a slash
if [[ "$WEBAPI_URL" != */ ]]; then
    WEBAPI_URL="$WEBAPI_URL/"
fi
```

This might need to be slightly modified to work with plain /bin/sh.

Reply:
Good catch. I've added a POSIX-shell `case` statement instead.